app/src/main/java/com/example/concordia_campus_guide/Fragments/CalendarFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.example.concordia_campus_guide.Fragments;
+
+import android.database.Cursor;
+import androidx.core.app.ActivityCompat;
+import androidx.core.content.ContextCompat;
+import androidx.lifecycle.ViewModelProviders;
+
+import android.Manifest;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.concordia_campus_guide.Models.CalendarEvent;
+import com.example.concordia_campus_guide.R;
+
+import java.util.ArrayList;
+
+public class CalendarFragment extends Fragment {
+
+    private CalendarViewModel mViewModel;
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.calendar_fragment, container, false);
```

Comment:
Do we need/have a view for this? I thought it was a system enable feature

Reply:
I made the view to see if the integration was working. It also makes it easier to implement #17 as the person will just need to add on top of this view.